style.css

```diff
@@ -3,11 +3,62 @@
 * {
   padding: 0;
   margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
 }
 
 html,
 body {
   overflow-x: hidden;
+} 
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1.2rem;
+  margin-left: 0.5rem;
+  justify-content:space-between;
+
```

Comment:
[Optional] Currently, when you navigate to other sections from the navbar, such as 'About' or 'Home,' you may notice that the navbar remains at the top. For a better user experience, you can set the header to a fixed position. Here is the code you can use as a reference for that purpose. 😁 

```
    position: fixed;
    top: 0;
    left: 0;
    width: 100%;
    background-color: white;
    // can remove the margin-left when you are using these
```